doc/feature/colorMaps.md

```diff
@@ -7,253 +7,102 @@ applications. An exemplary case is the transfer functions that map density to
 RGBA and opacity values in volume rendering with an emission-absorption optical
 model.
 
-This specification describes the messages and C++ helper classes to deal with
-color maps as a particular case of transfer function (TF). The more generic
-case will be addressed in a separate document.
+This specification describes the messages to deal with maps as a particular
+case of transfer function (TF). The more generic case will be addressed in a 
+separate document.
+
+This document follows the discussion initiated in by [https://github.com/HBPVIS/Lexis/pull/19]
 
 ## Requirements
 
-- Represent RGBA colormaps with an independent TF for each channel. The basic
+- Represent RGBA color maps with an independent TF for each channel. The basic
   TF definition should be reusable for the more generic case.
-- The C++ helper class will encapsulate a colormap and allow the conversion of
-  the network representation into a OpenGL ready lookup table for creating a
-  texture.
+- Represent extra material properties such as light emission, color contribution
+  and specular color.
```

Comment:
Well, nobody was really convinced, so I removed it. I am OK to adding it back, no opinion there. We can always add it later if any of the renderers needs/wants it.